src/containers/EventForm/components/ArtworkForm/function.js

```diff
@@ -0,0 +1,28 @@
+import { ArtworksRepository, PeopleRepository } from 'hackoss';
+import { FirebaseApp } from 'utils/firebase';
+
+async function getPeopleBase() {
+  const pplRepo = new PeopleRepository(FirebaseApp);
+  const people = await pplRepo.getPeople();
+  return people;
+}
+export async function getPeople() {
```

Comment:
Please correct me if I am wrong, but isn't this function just getting the names of the people? Then in that case would it be better to name this function as `getPeopleNames`?

Reply:
Yep. Sounds fair.